src/analysis/jump.ml

```diff
@@ -155,6 +188,10 @@ let get typed_tree pos target =
     else begin
       let nodes = skip_non_moving pos enclosings in
       let node = find_node preds nodes in
+      match target with 
+      | "match-next-case" -> find_case_pos (get_cases_from_match node) pos Next
+      | "match-prev-case" -> find_case_pos (List.rev (get_cases_from_match node)) pos Prev
```

Comment:
Nice trick 😃
I was puzzled for a moment as to why the comparison was working and then I saw that `List.rev`.

Reply:
Thank you 😃